kernel_config

```diff
@@ -1170,7 +1170,7 @@ CONFIG_NET_SCH_FIFO=y
 # CONFIG_DCB is not set
 CONFIG_DNS_RESOLVER=y
 # CONFIG_BATMAN_ADV is not set
-# CONFIG_OPENVSWITCH is not set
+CONFIG_OPENVSWITCH=m
```

Comment:
Thanks to @dduportal, preparation for future work on using openVSwitch to control container networking.